src/dependency-manager/package.json

```diff
@@ -17,13 +17,15 @@
   "homepage": "https://github.com/architect-team/architect-cli",
   "license": "GPL-3.0",
   "dependencies": {
+    "@types/mustache": "^4.0.1",
```

Comment:
Can you add @types to the dev dependencies?

Reply:
Done